packages/thumbprint-react/components/ServiceCard/index.jsx

```diff
@@ -50,9 +50,9 @@ function ServiceCardDescription({ iconColor, icon, children }) {
     );
 }
 
-export default function ServiceCard({ url, children }) {
+export default function ServiceCard({ url, onClick, target, children }) {
```

Comment:
Thoughts on calling the `target` prop `shouldOpenInNewTab`? A boolean may be better since we want to support `_blank` and the default `_self`, not `_parent` ot `_top`. (This is also consistent with the prop name in `Button`.)

Reply:
Good call. I found the `getAnchorProps` utility that was being used in `Plain` and decided that it would be a good idea to employ here as well so I did and pushed that up. Hope that was okay. The only side effect is that it also provides an `isDisabled` prop, but the behavior will be consistent between `ServiceCard` and `Plain`/`Link`, which I think is a net positive. Thoughts?